static/app/utils/discover/genericDiscoverQuery.tsx

```diff
@@ -77,6 +77,7 @@ type BaseDiscoverQueryProps = {
    * passed, but cursor will be ignored.
    */
   noPagination?: boolean;
+  options?: Omit<Parameters<typeof useQuery>[2], 'initialData'>;
```

Comment:
i think this should be `UseApiQueryOptions` exported from `sentry/utils/queryClient`

Reply:
nvm its not using useApiQuery

Reply:
The discover query uses `useQuery` instead of our own `useApiQuery` so that might actually not work. Could use `UseQueryOptions` exported from the lib though.

Reply:
just copying the usage of this type here: https://github.com/getsentry/sentry/blob/7d84829c2111305cd220b9ef1d482964b7bb45ad/static/app/utils/discover/genericDiscoverQuery.tsx#L133